content/community.html

```diff
@@ -30,6 +30,15 @@
     more elaborate or controversial changes, please discuss them on the mailing
     list before sending a pull request.
   </p>
+  <h1>Commercial</h1>
+  <p>
+    This is a list companies which provide supporting or consulting services for
+    Prometheus. Prometheus is an independent open source project which does not
+    endorse any company.
```

Comment:
missing "of". Could you also change "companies" -> "third-party companies" to add even more emphasis on this fact?